tests/pipeline/test_quarters_estimates.py

```diff
@@ -888,6 +900,12 @@ def create_expected_df(cls, tuples, end_date):
         df.index = df.index.rename('knowledge_date')
         df['at_date'] = end_date.tz_localize('utc')
         df = df.set_index(['at_date', df.index.tz_localize('utc')]).ffill()
+        new_sids = set(cls.get_sids()) - set(df.columns)
+        df = pd.concat([df, pd.DataFrame(columns=new_sids)])
+        # Have to explicitly cast to float so that nans are of type
+        # np.float64 - otherwise they won't compare equal.
+        for col in new_sids:
```

Comment:
I don't understand why the types are off:
```
In [1]: df = pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]}, dtype='f8')

In [2]: df.reindex(columns=['a', 'b', 'c'])
Out[2]: 
     a    b   c
0  1.0  4.0 NaN
1  2.0  5.0 NaN
2  3.0  6.0 NaN

In [3]: _.dtypes
Out[3]: 
a    float64
b    float64
c    float64
dtype: object
```
also, in the astype below, use `float64` or `float32` to be explicit.